<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanForLostFilesWidget</class>
 <widget class="QWidget" name="ScanForLostFilesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>590</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select folder:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KUrlRequester" name="reqFolder" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnExpandAll">
       <property name="text">
        <string>Expand All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCollapseAll">
       <property name="text">
        <string>Collapse All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnScanFolder">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowAllFiles">
     <property name="toolTip">
      <string>Open the torrents without asking any questions.</string>
     </property>
     <property name="text">
      <string>Show all files</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionOpen_file">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionDelete_on_disk">
   <property name="icon">
    <iconset theme="delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete on disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionCopy_to_clipboard">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
